lib/bbc/a11y/js/standards/exactlyOneMainLandmark.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  name: 'Exactly one main landmark',
+
+  validate: function($, fail) {
+    var mainLandmarks = $("[role='main']");
+    var count = mainLandmarks.length;
+    if (count == 0) {
+      fail('Found 0 elements with role="main".');
+    } else if (count > 1) {
```

Comment:
Actually there's just a small difference in the formatting that I'd like to keep (colon before selectors, otherwise full stop):
when there are no failures: `Found 0 elements with role="main".`
when there are failures: `Found <count> elements with role="main": <selectors>`